File: Mobile_flutter/clinic/lib/models/clinic.dart
class Clinic {
  final int id;
  final String name;
  final String description;
  final String contactEmail;
  final String contactNumber;

  Clinic({
    required this.id,
    required this.name,
    required this.description,
    required this.contactEmail,
    required this.contactNumber,
  });

  factory Clinic.fromJson(Map<String, dynamic> json) {
    return Clinic(
      id: json['id'],
      name: json['name'],
      description: json['description'],
      contactEmail: json['contactEmail'],
      contactNumber: json['contactNumber'],
    );
  }
}

File: Backend_api/Przychodnia/Przychodnia/Entities/Clinic.cs
namespace Przychodnia.Entities
{
    public class Clinic
    {

        public int Id { get; set; }
        public string Name {get; set;}
        public string Description { get; set;}
        public string ContactEmail { get; set;}
        public string ContactNumber { get; set;}

        //address
        public string City { get; set; }
        public string Street { get; set; }
        public string BuildingNumber { get; set; }
        public string PostalCode { get; set; }

    }
}
